addons/van_jsx/src/jsx-runtime.js

```diff
@@ -0,0 +1 @@
+export * from "./index";
```

Comment:
This can be fixed by adjusting the configuration of package.json.

Reply:
Adding this file may cause the vscode editor's smart import to carry a special path.

For example

```
import {} from "van_jsx/jsx_runtime";
```